mmpose/models/detectors/pose_lifter.py

```diff
@@ -20,6 +20,11 @@ def __init__(self,
                  backbone,
                  neck=None,
                  keypoint_head=None,
+                 traj_backbone=None,
+                 traj_neck=None,
+                 traj_head=None,
+                 loss_semi=None,
+                 warmup_epochs=0,
```

Comment:
I think it's better to move warmup_epochs to the arguments of semi_loss.

Reply:
The loss function is called every iteration. I have no idea how to get the number of iterations in an epoch. It can be changed due to different batch sizes?